### PR TITLE
Deprecate --pbs-runner

### PR DESCRIPTION
**Pull request by mih**
Fixes datalad/datalad#5938

### CODE REVIEW

File: datalad/cmdline/helpers.py [hunk 1]
```diff
@@ -313,6 +313,10 @@ def strip_arg_from_argv(args, value, opt_names):
 
 
 def run_via_pbs(args, pbs):
+    warnings.warn("Job submission via --pbs-runner is deprecated."
+                  "Use something like condor_run",
```
Comment (yarikoptic) [inline datalad/cmdline/helpers.py:317]:
```suggestion
                  "Use something like condor_run from datalad-htcondor extension",
```

Comment (mih) [reply datalad/cmdline/helpers.py:317]: Err, no. I meant the standard HTCondor command. I do not want to advertise `datalad-htcondor`.

Comment (yarikoptic) [reply datalad/cmdline/helpers.py:317]: ah, misunderstood. ok

File: datalad/cmdline/common_args.py [hunk 2]
```diff
@@ -50,7 +50,7 @@
     'pbs-runner', ('--pbs-runner',),
     dict(choices=['condor'],
          default=None,
-         help="""execute command by scheduling it via available PBS.  For settings, config file will be consulted""")
+         help="""DEPRECATED, will be removed in a future release.""")
```
Comment (yarikoptic) [inline datalad/cmdline/common_args.py:53]: useful would have been to add a comment with version (0.16) when it is ok to remove, or someone would need to do archaeological expedition to figure out for sure if it is ok time to remove

don't want to hold this PR with this, so let's proceed

Comment (mih) [reply datalad/cmdline/common_args.py:53]: Filed https://github.com/datalad/datalad/issues/5962 with milestone 0.16

**Pull request merged**